### PR TITLE
2021-2022 fall update

### PR DESCRIPTION
**Pull request by DavidPurdy1**
fixed website for mobile devices, updated pictures/info/hares, refactored a little bit.

In the future would like to get rid of the embedded calendar which has a lot of dependencies and replace it with easy to change image of the rush calendar because it never gets updated otherwise. Could also add links to the rush page like the linktree link we have. Also not familiar with materialize.css, but why do we have 2 of them in the project?

### CODE REVIEW

**Comment by schwalbs** [comment]
> refactored a little bit

> touches 97 files

**Comment by schwalbs** [comment]
> Also not familiar with materialize.css, but why do we have 2 of them in the project?

I don't remember, this was my first web project I created from scratch back in 2015 I think. That CSS library is way outdated. One is the minified I believe and the other is the full file.

File: js/site-components.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+/* Using custom elements to reduce code duplication
+    -On re-election change contact links in footer
+    -Add links to the hare in the header
+*/
+class Header extends HTMLElement {
```
Comment (schwalbs) [inline js/site-components.js:5]: Nice

File: index.html [hunk 2]
```diff
@@ -4,112 +4,57 @@
...
+          improvement. Beyond all of the direct academic support we give our brothers, having a group of men that you
+          know and trust to support you in your courses allows you to be a more successful student.</p>
+        <p>Thank you for your time! If you have any questions about Delta Sigma Phi or Greek life in general please feel
+          free to <a class="green-text text-accent-4" href="mailto:bissmepj@rose-hulman.edu">reach out to me</a> at any
+          time!</p>
         <p>Sincerely,</p>
-        <br />
-        <img src="img/index/signature-bissmepj.png" alt="Phil Bissmeyer" />
+        <br>
+        <img src="img/index/signature-bissmepj.png" alt="Phil Bissmeyer" width="300px" style="margin-left: -50px;" />
```
Comment (schwalbs) [inline index.html]: Do we have that new signature?

File: index.html [hunk 3]
```diff
@@ -4,112 +4,57 @@
...
+          have. Our brothers are passionate about video games, cars, intramural and Rose-Hulman athletics, movies and a
+          multitude of other hobbies. What makes Delta Sigma Phi unique is that these diverse interests don’t divide us,
+          instead we celebrate the passion that our brothers have for their hobbies.</p>
+        <p>Students come to Rose-Hulman to be the best engineer they can be, and as a chapter we do everything we can to
+          support our brothers and their academic success. We schedule group study sessions, establish upperclassmen
+          tutors for brothers who are struggling with a course, and give out quarterly awards for academic success and
+          improvement. Beyond all of the direct academic support we give our brothers, having a group of men that you
+          know and trust to support you in your courses allows you to be a more successful student.</p>
+        <p>Thank you for your time! If you have any questions about Delta Sigma Phi or Greek life in general please feel
+          free to <a class="green-text text-accent-4" href="mailto:bissmepj@rose-hulman.edu">reach out to me</a> at any
```
Comment (schwalbs) [inline index.html]: Pretty sure he's not the president anymore

File: history.html [hunk 4]
```diff
@@ -140,46 +116,17 @@ <h2 class="header">Our Chapter</h2>
...
-    </div>
-  </footer>
-
+  <site-footer></site-footer>
   <script type="text/javascript" src="js/jquery.min.js"></script>
   <script type="text/javascript" src="js/materialize.min.js"></script>
+  <script type="text/javascript" src="js/site-components.js"></script>
   <script type="text/javascript" src="js/base.js"></script>
+  <script>
+    if (window.matchMedia("(max-width: 480px)").matches) {
```
Comment (schwalbs) [inline history.html]: We should do this with a CSS class

```css
@media all and (min-width: 480px){
  .hidden-tablet-up {
    display: none;
  }
}
```

**Review by schwalbs — CHANGES_REQUESTED** [review]

**Comment by DavidPurdy1** [comment]
> > refactored a little bit
> 
> > touches 97 files

haha yeah she needed some work

**Comment by yorker78** [comment]
I'm afk till this weekend, I'll take a look then. Don't feel like you NEED approval from us, but if you want other eyes on it, I'll be happy to once I'm able.

**Comment by DavidPurdy1** [comment]
Okay, cool thanks. Didn't want to just come in a change a whole bunch of things on my first PR here. The last thing I am going to do is probably change the rush calendar to just be jpg instead of the interactive calendar. Once I finish that I might merge it in.

**Comment by yorker78** [comment]
If you're webmaster, then you're the primary maintainer and can do as you please (just don't fuck it up too much), otherwise they're the person to get approval from. The alumni are just here as an extra set of eyes and experience to reach out to for assistance. I'm not trying to downplay our contributions, but at the end of the day it's a website by & for actives. Also if you have an idea and you're unsure how to implement, you can try creating an issue and assigning it (no guarantee it'll actually get picked up by the person you've assigned it to).

File: brothers.html [hunk 5]
```diff
@@ -8,7 +8,7 @@
     content="Delta Sigma Phi Fraternity | Zeta Lambda Chapter | Rose-Hulman Institute of Technology">
 
   <link rel="shortcut icon" href="ico/favicon.ico">
-  <link type="text/css" rel="stylesheet" href="css/materialize.css" media="screen,projection" />
+  <link type="text/css" rel="stylesheet" href="css/materialize.min.css" media="screen,projection" />
```
Comment (schwalbs) [inline brothers.html:11]: We're yeeting `css/materialize.min.css`, does it get auto-generated?

**Comment by schwalbs** [comment]
Like Billy said, go bonkers in this repo, it's all yours. I won't feel bad if you make my crappy code better. Go for a full rewrite if you want, there's a lot that can be improved on here. More templating and a newer CSS framework would really improve it.

I'm fairly active on Github for work and would be happy to review code or help with changes.

File: brothers.html (hunk 5 above)
Comment (cocicat) [reply brothers.html:11]: It seems like its not getting yeeted, but rather condensed all into line six for viewing sake. Is this accurate @DavidPurdy1 ?

Comment (schwalbs) [reply brothers.html:11]: oh you right. I assumed 7500 lines of deletion was a full yeet.

**Review by cocicat — APPROVED** [review]
Fantastic changes, especially replacing the site-header and site-footer. Thanks David.

**Pull request merged**